app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -151,12 +138,16 @@ def identity_proofing_requested_for_verified_user?
   end
 
   def resolved_authn_context_result
-    @resolved_authn_context_result ||= AuthnContextResolver.new(
+    authn_context_resolver.resolve
```

Comment:
not a huge deal, but part of the reason for memoization is to make an application more efficient. the `resolve` method is doing the bulk of the work. did you consider memoizing `resolved_authn_context_result` along with `authn_context_resolver` as it's called multiple times?

Reply:
I considered it. When `vtr` is not present, which is most of the time, it would be using `acr_result` which is memoized.